units/debug-shell.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-debug-generator(8)
 DefaultDependencies=no
 IgnoreOnIsolate=yes
 ConditionPathExists={{DEBUGTTY}}
+After=systemd-vconsole-setup.service
```

Comment:
This isn't really about resolutions. The problem is that if loadkeys is interrupted, the keymap might not be loaded at all and the shell might be unusable. I think that realistically `systemd-vconsole-setup` is very quick, so I don't think this delay matters much. If it turns out that it does, we can always drop this.

Reply:
why would debug-shell interrupt loadkeys?

Reply:
Because of TTYVHangup=yes. The systemd does a TIOCVHANGUP while loadkeys running and loadkeys loses the terminal. All subsequent ioctls that loadkeys does complete with `EIO`.

See https://github.com/legionus/kbd/issues/92#issuecomment-1554451788

Reply:
This is what the whole bug is about: units that have `TTYVHangup=yes`, when started, at the same time (or maybe right after?) systemd-vconsole-setup, break `loadkeys`. The commit describes this, and https://github.com/systemd/systemd/issues/26908 and https://github.com/legionus/kbd/issues/92 have the details.